curriculum/js1/10.js

```diff
@@ -7,7 +7,9 @@
  * @param {function} c
  */
 
-const solution = (a, b, c) => {
+const solution = (a, b, C) => {
+  setTimeout(C, a*1000);
+  setTimeout(C, (a+b)*1000);
```

Comment:
I will approve this. But I have a comment on the comment. The comment on line 7 says that the third parameter is `c`, but the parameter is actually named `C`. There's nothing illegal about using an uppercase variable name, but the convention is to start the name with a lowercase letter. Generally, names that start with an uppercase letter are class-names or constructors.

Reply:
Got it. Thanks!